kpops/component_handlers/kafka_connect/connect_wrapper.py

```diff
@@ -121,16 +121,26 @@ def update_connector_config(
             self.update_connector_config(connector_name, kafka_connect_config)
         raise KafkaConnectError(response)
 
+    def get_connector_config(
+        self, connector_name: str, config: KafkaConnectConfig
+    ) -> dict:
+        connector_config = config.dict(exclude_none=True, exclude_unset=True)
+        if (name := connector_config.get("name")) and name != connector_name:
+            raise ValueError("App config name should be the same as component name")
```

Comment:
Is `App config name` referring to the Kafka Connect config?
If so it is a bit misleading and I would change it to `Kafka Connect config`.

Reply:
```suggestion
            raise ValueError("Connector name should be the same as component name")
```